### PR TITLE
added user guide-under splitting section-2 commands 

### PR DESCRIPTION
**Pull request by WillySeahh**
Added commands under split functionality (calculate and list food)

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -127,6 +127,26 @@ Example: +
...
+
+Format: +
+`calculate [-n NAME]. . .` +
+
+Example: +
+`calculate` +
+`calculate -n john`
+`calculate -n john -alice`
+
+==== List items  `list food`
```
Comment (sheexiong) [inline docs/UserGuide.adoc]: Should we call it List Food instead?

Comment (akhilvuputuri) [reply docs/UserGuide.adoc]: Yep agree, will do a PR after everyone else is done adding to make the changes unless Bob can change it while adding his part? @Jarrod-Bob

Comment (sheexiong) [reply docs/UserGuide.adoc]: Sure

Comment (WillySeahh) [reply docs/UserGuide.adoc]: Yup, understood.

**Pull request merged**